_docs/Qabel-Index.md

```diff
@@ -39,21 +39,43 @@ email
 : E-Mail address
 
 phone
-: Phone number (must be capable of receiving SMS)
+: Phone number (must be capable of receiving SMS). Incoming phone
+numbers are normalized to ITU-T E.164 according to the specified
+locale (via HTTP Accept-Language), if they are not already. It is
+recommended to normalize phone numbers on the client.
+
+: The server might reject phone numbers due to blacklisting of their
+country code.
```

Comment:
The ":" *should* be correct (no way to be sure with markdown tbh) and place this paragraph on the same definition-list level as the preceding paragraph.